Engine/Setup/BacktestingSetupHandler.cs

```diff
@@ -164,6 +164,7 @@ public bool Setup(IAlgorithm algorithm, IBrokerage brokerage, AlgorithmNodePacke
             if (algorithm == null)
             {
                 Errors.Add("Could not create instance of algorithm");
+                Errors.Add("Could not create instance of algorithm");
```

Comment:
Why is this line duplicated ? Hope it's not a bug of the clean whitespace plugin :)

Reply:
great catch!, missed that when diffing -- not sure, it might have been a fat finger ctrl+d (duplicate line)

Reply:
Actually, must have been, ctrl+s and ctrl+d are easy to fat finger :/